pkgs/by-name/pr/proton-ge-bin/package.nix

```diff
@@ -25,7 +25,14 @@ stdenvNoCC.mkDerivation (finalAttrs: {
     # Also leave some breadcrumbs in the file.
     echo "${finalAttrs.pname} should not be installed into environments. Please use programs.steam.extraCompatPackages instead." > $out
 
-    ln -s $src $steamcompattool
+    mkdir $steamcompattool
+    ln -s $src/* $steamcompattool
+    rm $steamcompattool/{compatibilitytool.vdf,proton,version}
+    cp $src/{compatibilitytool.vdf,proton,version} $steamcompattool
+
+    sed -i -r 's|GE-Proton[0-9]*-[0-9]*|GE-Proton|' $steamcompattool/compatibilitytool.vdf
+    sed -i -r 's|GE-Proton[0-9]*-[0-9]*|GE-Proton|' $steamcompattool/proton
+    sed -i -r 's|GE-Proton[0-9]*-[0-9]*|GE-Proton|' $steamcompattool/version
```

Comment:
I tested modifying the version in `$steamcompattool/version`, and it doesn't seem to invalidate the currently selected proton version.

So this line seems unnecessary; moreover, MangoHud can display the currently active proton version by reading it from `$steamcompattool/version`. With this line it would still indicate that proton-ge is used but not it's version.

Reply:
I think that merits keeping the full version information in `$steamcompattool/version`, as I have only added the modification to it to preserve internal consistency.